tests/functional/resources/user_fields-D8.php

```diff
@@ -15,7 +15,7 @@
 
 // Create a user.
 $values = [
-  'field_user_email' => 'joe.user.alt@myhome.com',
+  'field_user_email' => $extra[1],
```

Comment:
Where does $extra come from?

Reply:
It corresponds to the way the script is called. Please see https://github.com/miloskroulik/drush/blob/879c46c8ec9704744cd08266765f0f34ebf263b7/tests/functional/SqlSyncTest.php#L127